content_scripts/main.js

```diff
@@ -145,7 +152,7 @@ function shouldLinkParents(node) {
 
 // Returns true if we should link the node.
 function shouldLink(node) {
-  if (EXCLUDED_TAGS.hasOwnProperty(node.tagName) || isNodeEditable(node)) {
+  if (EXCLUDED_TAGS.hasOwnProperty(node.tagName) && EXCLUDED_TAGS[node.tagName] || isNodeEditable(node)) {

```

Comment:
Now that we're actually using the true/false value inside EXCLUDED_TAGS, let's just get rid of the "hasOwnProperty" check. If the tag we're checking isn't in EXCLUDED_TAGS, it should return undefined anyways, which is fine.